src/components/BetweenTwoCastles/index.jsx

```diff
@@ -1,55 +1,51 @@
 import React from "react";
 import Resource from "components/Resource";
+import DSixInput from "components/DSixInput";
 import "./BetweenTwoCastles.scss";
 import woodDoorSrc from "images/wooden-door.png";
 import stoneDoorSrc from "images/stone-door.png";
 
-const BetweenTwoCastles = () => {
-  const renderInput = (location) => (
-    <input
-      key={location}
-      className={location}
-      type="number"
-      min="1"
-      max="6"
-    />
-  );
+const BetweenTwoCastles = () => (
+  <>
+    <p>
+      Fill squares from the bottom up; #s on top must be lower than
+      those below. When you complete a column, gain a bonus. Score 1
+      star per completed row (each castle has 3 rows).
+    </p>
+    <div className="game">
+      {/* Row 1 */}
+      <b />
```

Comment:
I know this is not the point, but what are all of these `<b />` tags for?

Reply:
I can't remember exactly why, but I think I read somewhere on the internet that it's a legit way to work with the need for an empty CSS Grid cell. It's been a while though, so not totally sure. I think it can make it a little easier to figure out which column an element will be placed in than specifying it in CSS rules.

Reply:
Interesting. They're bold tags with no content. I wonder what about this tag makes it effective: my guess is that it's short and that a number of tags would work. Have you seen grid template areas? Super neat https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas

I'm nearly sure that these `b` tags are not accessible, but I don't think that needs to be a concern of this project.